cvat/schema.yml

```diff
@@ -5127,6 +5672,43 @@ components:
           format: binary
       required:
       - annotation_file
+    AnnotationId:
+      type: object
+      properties:
+        obj_id:
+          type: integer
+          readOnly: true
+        job_id:
+          type: integer
+          readOnly: true
+        type:
+          allOf:
+          - $ref: '#/components/schemas/AnnotationIdTypeEnum'
+          readOnly: true
+    AnnotationIdTypeEnum:
+      enum:
+      - tag
+      - track
+      - rectangle
+      - polygon
+      - polyline
+      - points
+      - ellipse
+      - cuboid
+      - mask
+      - skeleton
```

Comment:
This seems to largely overlap with `ShapeType`. Perhaps it would be better to replace the shape types in this enum with a single `shape` member, and add an additional `shape_type` field (of type `ShapeType`) to `AnnotationId`?

Reply:
I've split it into 2 fields - `type` and `shape_type`.